core/src/main/java/io/questdb/mp/Job.java

```diff
@@ -26,6 +26,7 @@
 
 import org.jetbrains.annotations.NotNull;
 
+@FunctionalInterface
```

Comment:
why, there is no downside i guess, just looks like a random change

Reply:
Yeah, just noticed this was used as a functional interface, so I added it for clarity. Might help some IDE features.